onnxruntime/core/framework/session_state.cc

```diff
@@ -1046,7 +1046,7 @@ Status SessionState::CreateSubgraphSessionState() {
       auto subgraph_session_state =
           std::make_unique<SessionState>(*subgraph, execution_providers_,
                                          thread_pool_, inter_op_thread_pool_, data_transfer_mgr_,
-                                         logger_, profiler_, sess_options_, nullptr, allocators_);
+                                         logger_, profiler_, sess_options_, nullptr, allocators_, prepacked_weights_container_);
```

Comment:
Btw- I think the fix is to put `prepacked_weights_container_` in place of the `nullptr`

Reply:
Thanks, I noticed that yesterday as well - just hadn't pushed yet!